.werft/util/kubectl.ts

```diff
@@ -134,7 +134,7 @@ export function createNamespace(namespace: string, kubeconfig: string, shellOpts
 
     // (re-)create namespace
     [
-        `kubectl --kubeconfig ${kubeconfig} create namespace ${namespace}`,
+        `kubectl --kubeconfig ${kubeconfig} create namespace ${namespace} --dry-run=client -o yaml | kubectl --kubeconfig ${kubeconfig} apply -f -`,
```

Comment:
Interesting pattern. You're generating the YAML before you apply this. Is this something we should keep in the code because there are advantages or is this more like a temporary debug thing and we should remove it for the sake of simplicity?